src/advanced/components/pages/admin/AdminPage.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useAtom } from 'jotai';
+import { activeTabAtom } from '../../../atoms/uiAtoms';
+import { AdminHeader } from './AdminHeader';
+import { ProductManagement } from './ProductManagement';
+import { CouponManagement } from './CouponManagement';
+import { ProductWithUI } from '../../../types';
+
+interface AdminPageProps {
+  onSetActiveTab: (tab: 'products' | 'coupons') => void;
+  onAddProduct: (product: any) => void;
+  onUpdateProduct: (productId: string, updates: any) => void;
+  onDeleteProduct: (productId: string) => void;
+  onStartEditProduct: (product: ProductWithUI) => void;
+}
+
+export const AdminPage: React.FC<AdminPageProps> = ({
+  onAddProduct,
+  onUpdateProduct,
+  onDeleteProduct,
+  onStartEditProduct,
+}) => {
+  const [activeTab] = useAtom(activeTabAtom);
+
+  return (
+    <div className='max-w-6xl mx-auto'>
+      <AdminHeader />
+
+      {activeTab === 'products' ? (
+        <ProductManagement
+          onAddProduct={onAddProduct}
+          onUpdateProduct={onUpdateProduct}
+          onDeleteProduct={onDeleteProduct}
+          onStartEditProduct={onStartEditProduct}
+        />
+      ) : (
+        <CouponManagement />
+      )}
```

Comment:
```suggestion
      {{
        products: <ProductManagement
          onAddProduct={onAddProduct}
          onUpdateProduct={onUpdateProduct}
          onDeleteProduct={onDeleteProduct}
          onStartEditProduct={onStartEditProduct}
        />,
        coupons:<CouponManagement />
      }[activeTab]}
```

이렇게 작성해도 좋을 것 같아요 탭이 늘어났을 때 변경이 쉽도록

Reply:
지훈님이 말씀하신대로 한다면 이 객체를 따로 변수로 분리하면 좋겠네요


```suggestion
type TabType = 'products' | 'coupons'

const activeSection: Record<TabType, ReactNode > = {
  products: <ProductManagement
    onAddProduct={onAddProduct}
    onUpdateProduct={onUpdateProduct}
    onDeleteProduct={onDeleteProduct}
    onStartEditProduct={onStartEditProduct}
  />,
  coupons: <CouponManagement />
};
```

Reply:
변수로 분리하면 읽는데 왔다갔다해야하니까 이건 어떤가요

```
      <Switch
          value={activeTab}
          cases={{
            products: <ProductManagement
              onAddProduct={onAddProduct}
              onUpdateProduct={onUpdateProduct}
              onDeleteProduct={onDeleteProduct}
              onStartEditProduct={onStartEditProduct}
            />,
            coupons:<CouponManagement />
        }}
      />
```